iron-location.html

```diff
@@ -53,6 +53,41 @@
   (function() {
     'use strict';
 
+    var workingURL;
+
+    var urlDoc;
+
+    /**
+     * @type {string} path
+     * @type {string=} base
+     * @return {URL|HTMLAnchorElement}
+     */
+    function resolveURL(path, base) {
+      var url;
+      if (workingURL === undefined) {
+        workingURL = false;
+        try {
+          var u = new URL('b', 'http://a');
```

Comment:
If you look at `/test/url-import.html` we add an extra check to see if

`new URL('http://www.google.com/?foo bar').href == 'http://www.google.com/?foo%20bar';`

This is because of a wrong implementation in edge / IE that doesn't encode spaces in queries unlike every other browser and polyfill. This should fix almost all of the Edge test failures.